src/mixins/__tests__/widget.test.js

```diff
@@ -20,17 +21,17 @@ const createFakeIndexWidget = () => ({
 describe('on root index', () => {
   it('adds a widget on create', () => {
     const instance = createFakeInstance();
-    const Test = createFakeComponent();
-
     const widget = { render: () => {} };
     const factory = jest.fn(() => widget);
     const connector = jest.fn(() => factory);
+    const Test = createFakeComponent({
```

Comment:
I moved the widget mixin inside the Test component, because I think it makes more sense than putting it in the `mount()` call. (And this works and the other way doesn't)